orderbook/src/main.rs

```diff
@@ -72,6 +74,20 @@ pub async fn orderbook_maintenance(
     unreachable!()
 }
 
+pub async fn database_metrics(metrics: Arc<Metrics>, database: Database) -> ! {
```

Comment:
Is there a specific reason you don't want to do this as part of orderbook maintenance? I think the less different "tasks" we spawn the easier it will be to reason about what is happening in the system (e.g. race conditions won't happen in a sequential setup). Of course, if there is a good reason in priority or frequency it might make sense but I personally don't see this here.